include/secure_services.h

```diff
@@ -15,24 +28,29 @@ extern "C" {
 #endif
 
 /** Request a system reboot from the Secure Firmware.
+ *
+ * Rebooting is not available from the Non-Secure Firmware.
+ *
+ * @note Usually, @c sys_reboot should be called instead of calling this
```

Comment:
I still do not understand this note @oyvindronningstad

Reply:
Sorry! I meant to remove it. Should be done now.